Running cluster for deep fits
120073 120074 normal
Running object 120073
Not enough g observations!
WARNING: Could not memory map array with mode='readonly', falling back to mode='denywrite', which means that the array will be read-only [astropy.io.fits.file]
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  49
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  37
Min gap is: 7.046875
Number of observations now:  10
Counts: {'r': 49, 'i': 34, 'z': 7}
Mu: {'g': 0.0, 'r': 0.15433423, 'i': 0.24898633, 'z': 0.396453}
No handles with labels found to put in legend.
Variable??
0
0
Non-Variable source in g
Variable??
0.023347317706793547
1.9209981721575806
Non-Variable source in r
Variable??
0.04147004021662585
1.083074828256001
Non-Variable source in i
Variable??
0.09101351308100614
0.8319023294723361
Non-Variable source in z
Variable:    0/4
Var Bands:   
Slope:       [1, 1, 1, 1]
Slope Error: [0.0, 0, 0.28010976915846975, 1.3231279517546988]
intercepts:  [0, 0, -0.016041579971206588, 0.058829927927010806]
Int Error:   [0.0, 0, 0.05898577792877158, 0.20404950587664292]
StDev:       [0.0, 0.058928885, 0.056629047, 0.089713916]
Not variable, closing plots & going to next row
Successfully finished!
Rows 120073 to 120074 ran in 2.7714 seconds
